scanner/enricher/nvd/nvd.go

```diff
@@ -345,14 +360,14 @@ func (e *Enricher) ParseEnrichment(ctx context.Context, rc io.ReadCloser) ([]dri
 	}()
 	var err error
 	dec := json.NewDecoder(rc)
-	ret := make([]driver.EnrichmentRecord, 0, 1024) // Wild guess at initial capacity.
+	ret := make([]driver.EnrichmentRecord, 0, 250_000) // Wild guess at initial capacity.
 	// This is going to allocate like mad, hold onto your butts.
 	for err == nil {
 		ret = append(ret, driver.EnrichmentRecord{})
 		err = dec.Decode(&ret[len(ret)-1])
 	}
 	zlog.Debug(ctx).
-		Int("count", len(ret)).
+		Int("count", len(ret)-1).
```

Comment:
note `ret` will have a length of 1 + number of enrichments because we allocated space in it and then try to decode. So on the last one, we allocate, then the decoding fails